src/walnut.ts

```diff
@@ -255,6 +260,27 @@ const hallways: Hallway[] = [
     new Room("2218"),
   ]),
 
+  // hallway 2401
+  new Hallway([
+    new Fork(BACK, ConnectionNode.C2401_T0_2100S, "the 2100s"),
+    new Room("2401", LEFT), // could be althelic director office
```

Comment:
If you want to add an alias to make 2401 the athletic director's office, you add an alias, just like there is for the Junior High Gym: add a third argument that says {aliases: ["Athletic Director's Office"]}

Reply:
I think he means that he thinks its the athletic director's office, but he's not sure

Reply:
I think someone told me 2401 is the athletic director's office, and I wrote that it is in #30, so let's just assume it is for now.